File: 태성/예산.py
d = [2,2,3,3]

budget = 10

def solution(d, budget):
    answer = 0
    d.sort()
    print(d)
    cnt = 0
    for a in d:
        if budget - a >= 0:
            cnt += 1
            budget = budget - a

    answer = cnt


    return answer

print(solution(d, budget))

File: 준영/프로그래머스/LV1/예산.js
function solution(d, budget) {
  var answer = 0;
  d.sort(function(a,b) {
    return a - b
  })

  for (let i of d) {
    if(budget >= i){
      budget-=i
      answer+=1
    }
  }
  return answer;
}